pyeed/network/network.py

```diff
@@ -51,6 +51,16 @@ class Config:
         description="List of selected sequences",
     )
 
+    data_Edge: Optional[dict] = Field(
+        default=None,
+        description="Dictionary with edge data",
+    )
+
+    naming_dict: Optional[dict] = Field(
```

Comment:
I think we should not add a `naming_dict` the `SequenceRecord` already has a `name` field for an arbitrary sequence name.

Reply:
Hmmm but the naming dic might be something different, in my case the naming dic is TEM-1, .... and not the name of the sequence record because that is not a common standard

Reply:
I suggest to overwrite the name, since the name from NCBI or UniProt is meaningless. You can also use the Antibiotic Resistance Ontology and add the corresponding reference to the ProteinRecord object. For instance this one: https://www.ebi.ac.uk/ols4/ontologies/aro/classes/http%253A%252F%252Fpurl.obolibrary.org%252Fobo%252FARO_3000876?lang=en